clang/test/SemaSYCL/intel-max-global-work-dim-device.cpp

```diff
@@ -62,8 +62,8 @@ struct TRIFuncObjGood1 {
 
 struct TRIFuncObjGood2 {
   [[intel::max_global_work_dim(3)]]
-  [[intel::max_work_group_size(8, 1, 1)]]
-  [[sycl::reqd_work_group_size(4, 1, 1)]] void
+  [[intel::max_work_group_size(8, 1, 8)]]
```

Comment:
Can you explain why you made this change?

Reply:
unrelated change. removed